content/news/030/index.md

```diff
@@ -114,6 +114,35 @@ fixes and additions:
 [Vangers]: https://www.gog.com/en/game/vangers
 [@kvark]: https://github.com/kvark/
 
+### [Way of Rhea][wor]
+
+![way of rhea capsule image](wor-capsule.jpg)
+
+[Way of Rhea][wor] is a puzzle adventure with hard puzzles and forgiving
+mechanics being produced by [@masonremaley][wor-mason-remaley] in a custom Rust
+engine. It has a demo available [on Steam][wor].
+
+Latest developments:
+
+- [Making Your Game Go Fast by Asking Windows Nicely][wor-fast] was published,
+discussing Windows specific performance tweaks in Way of Rhea's engine
```

Comment:
could you use two spaces for indenting long list lines, please

Reply:
Done! :)